pkgs/top-level/aliases.nix

```diff
@@ -538,6 +540,7 @@ mapAliases ({
 
   ### I ###
 
+  idrisPackages.sdl = throw "'name' has been removed, as it was broken and unmaintained"; # added 2024-05-09
```

Comment:
Does this really work?

Reply:
oh you're right, hum i am not sure this is an oversight. I am not sure how to set it then, any idea?

Reply:
> Does this really work?

I assume not, since things like `pkgs/top-level/python-aliases.nix` exist. I haven't time to dig into the impl atm

Reply:
![image](https://github.com/NixOS/nixpkgs/assets/53050011/06564770-b1d4-4074-bec6-a7f1631a5f3a)